release_notes.md

```
@@ -0,0 +1,79 @@
+# Release Notes
+
+## Press Enter to Sign up or Login #226
+ - Created by: parthmanhas
+ - PR Link: https://github.com/bluewave-labs/bluewave-onboarding/pull/230
+ - Merged on: 2024-09-24
+ - Linked Issues:
+   - Press Enter to Sign up or Login: https://github.com/bluewave-labs/bluewave-onboarding/issues/226
+
+## further code rabbit changes
+ - Created by: erenfn
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)